components/elm/src/biogeochem/AnnualUpdateMod.F90

```diff
@@ -32,7 +32,7 @@ subroutine AnnualUpdate(bounds, &
     !
     ! !USES:
     use clm_time_manager, only: get_step_size, get_days_per_year
```

Comment:
Just curious. Why is clm_time_manager not renamed?

Reply:
@tanzeli1982 The `clm_time_manager` is not changed in this PR because it breaks compatibility with FATES. @rgknox will be issuing a FATES PR that will allow `clm_time_manger` to be renamed as `elm_time_manager`.